go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/nyarly/spies v0.0.0-20200413230442-112961b2b018 // indirect
```

Comment:
Is this intended? Why does removing Knative add a new dep?

Reply:
No, it left after debug. I should remove it.

Reply:
Fixed it, see commit 577c562e5c2def8fb5d9c8a04b906f1ed59ba857.